test/test_master_controller.py

```diff
@@ -1010,6 +1010,7 @@ async def request(
     async def test_product_configure_reuse_name(self, client: aiokatcp.Client) -> None:
         await client.request("product-configure", "product", CONFIG_CBF_ONLY)
         await client.request("product-deconfigure", "product")
+        await asyncio.sleep(1)
```

Comment:
Remove this: the test is specifically to ensure that CBF products don't linger in the background after deconfigure the way SDP products do.

I know what's causing this - I'll make a comment elsewhere.

Reply:
Apologies! I meant to make a comment here/on the ticket to clarify why I'd done this. Just seen your other comment and will update accordingly. Thanks for the pointer.